patches/chrome-android-java-AndroidManifest.xml.patch

```diff
@@ -54,7 +54,7 @@ index fba6543bde641c1e29cbac01914000c75495cf2b..da9879111318168f381ffc3727b73a35
  
 +<!--
      <uses-permission android:name="com.google.android.apps.now.CURRENT_ACCOUNT_ACCESS" />
-+--> <uses-permission android:name="com.google.android.finsky.permission.BIND_GET_INSTALL_REFERRER_SERVICE"/> <uses-permission android:name="android.permission.READ_SYNC_SETTINGS"/>
++--> <uses-permission android:name="com.google.android.finsky.permission.BIND_GET_INSTALL_REFERRER_SERVICE"/>
```

Comment:
can't we get rid of this line entirely if we put this permission inside `{% include "//brave/android/java/AndroidManifest_user_permissions.xml" %}`